lib/psrand.h

```diff
@@ -34,9 +34,7 @@
  * $NoKeywords: $
  */
 
-#undef RAND_MAX
```

Comment:
This file seems to be missing include guards

Reply:
I approve these changes, the original file didn't have the guard either.